src/box/sql/sqlInt.h

```diff
@@ -2625,9 +2616,18 @@ expr_new_variable(struct Parse *parse, const struct Token *spec,
 void sqlExprListSetSortOrder(ExprList *, enum sort_order sort_order);
 
 void sqlExprListSetName(Parse *, ExprList *, Token *, int);
-void sqlExprListSetSpan(Parse *, ExprList *, ExprSpan *);
+
+/**
+ * Set the ExprList.a[].zSpan element of the most recently added item on the
+ * expression list.
+ *
+ * @param pList List to which to add the span.
+ * @param The span to be added.
```

Comment:
You missed the argument name here. I suggest to drop the doxygen comments liks this one. They are obvious anyway.

Reply:
Fixed. Other similar comments seem more helpful, so I've left them for now.